Rakefile

```diff
@@ -0,0 +1,7 @@
+require 'jasmine'
+
+load 'jasmine/tasks/jasmine.rake'
+
+ENV['JASMINE_CONFIG_PATH'] = 'spec/support/jasmine.yml'
```

Comment:
Should this be `ENV['JASMINE_CONFIG_PATH'] ||= 'spec/support/jasmine.yml'` so that it can be configured elsewhere if required?